ansible/roles/network-interfaces/tasks/main.yml

```diff
@@ -1,4 +1,18 @@
 ---
+
+- name: Check if NetworkManager package is installed
+  command: dpkg -l network-manager
+  register: nm_installed_check
+  failed_when: nm_installed_check.rc > 1
+
+- name: Stop NetworkManager
+  service: name=network-manager state=stopped enabled=no
```

Comment:
Logic looks good. Can you update to maintain style, please? e.g.

```
name: Stop NetworkManager
service:
  name: network-manager
  state: stopped
  enabled: no
```

and further down:

```
name: Remove NetworkManager
apt:
  pkg: network-manager
  state: absent
```